.circleci/config.yml

```diff
@@ -439,55 +431,6 @@ jobs:
       - run: |
           docker build .
 
-  test_build_alpine:
```

Comment:
I don't think we test alpine in github 🤔

Reply:
you are right, we don't, because we only build wheels on linux, windows, and macos

I'll add this back in for good measure

Reply:
updated